### PR TITLE
Slightly refactored DI class

### PR DESCRIPTION
**Pull request by SidRoberts**
Hello!

* Type: code quality
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [-] I wrote some tests for this PR
- [-] I updated the CHANGELOG

Slight tidy up of some of the methods.

### CODE REVIEW

File: phalcon/Di.zep [hunk 1]
```diff
@@ -139,12 +136,10 @@ class Di implements DiInterface
      */
     public function attempt(string! name, definition, bool shared = false) -> <ServiceInterface> | bool
     {
-        var service;
-
         if !isset this->services[name] {
-            let service = new Service(definition, shared),
-                this->services[name] = service;
-            return service;
+            let this->services[name] = new Service(definition, shared);
```
Comment (dschissler) [inline phalcon/Di.zep]: This will make the code slightly slower.

Comment (SidRoberts) [reply phalcon/Di.zep]: The difference is minimal and I'd much rather the code be neater and less complex.

Comment (dschissler) [reply phalcon/Di.zep]: That is fine but it represents something of a policy change and so its should be talked about.

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @SidRoberts

**Pull request merged**